### ISSUE TITLE
meta: add "need position by" field to issue template, use milestones to capture deadlines

### ISSUE THREAD

**Issue by hober** [WebKit/standards-positions#25]

**Comment by hober** [comment]
We now have milestones, but we're just using them to track cases where we've tentatively identified a position and we're waiting a week before adding the position label & closing the issue.

@annevk @marcoscaceres Should we also ask for some kind of "response appreciated by" date field to the new issue template?

**Comment by marcoscaceres** [comment]
It wouldn't hurt to ask. We should maybe make some kind of realistic recommendation though ("please allow 4-6 weeks" or something like that)

**Comment by annevk** [comment]
Do we expect people to request something other than ASAP?

**Comment by marcoscaceres** [comment]
yeah, maybe instead of asking for when, it should just say "We will try to respond ASAP, but give us 4-6 weeks to get back to you." or something like that.
